ext/openssl/ossl.c

```diff
@@ -739,15 +739,15 @@ ossl_crypto_fixed_length_secure_compare(VALUE dummy, VALUE str1, VALUE str2)
  * To sign a document, a cryptographically secure hash of the document is
  * computed first, which is then signed using the private key.
  *
- *   digest = OpenSSL::Digest::SHA256.new
+ *   digest = OpenSSL::Digest.new('SHA256')
  *   signature = key.sign digest, document
```

Comment:
Actually, the instantiation of OpenSSL::Digest is not even necessary and could be simplified as:

```ruby
signature = key.sign("sha256", document)
```

A change in 2007 (https://github.com/ruby/openssl/commit/7ff21742e1c29552ab6cdc4d8bb55257033e37e5#diff-95b444ed6d6ca9a03feb32bc391182d9) removed the restriction, but the document was not updated at that time.

Reply:
Thank you, I changed this.